include/flatbuffers/reflection_generated.h

```diff
@@ -87,7 +87,7 @@ inline const BaseType (&EnumValuesBaseType())[19] {
     Obj,
     Union,
     Array,
-    MaxBaseType
+    MaxBaseType,
```

Comment:
I don't think the trailing commas will work for c++11? Correct me if I'm wrong.

Reply:
I think it is supported 
https://godbolt.org/z/nKxnfjv1G